hx711_rpio.go

```diff
@@ -0,0 +1,128 @@
+// +build !windows,rpio
+
+package hx711
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/stianeikeland/go-rpio/v4"
+)
+
+// HostInit opens /dev/gpiomem. This needs to be done before Hx711 can be used.
+func HostInit() error {
+	return rpio.Open()
+}
+
+// NewHx711 creates new Hx711.
+// Make sure to set clockPin and dataPin to the correct pins.
+// https://cdn.sparkfun.com/datasheets/Sensors/ForceFlex/hx711_english.pdf
+func NewHx711(clockPinStr string, dataPinStr string) (*Hx711, error) {
```

Comment:
Maybe change this to clockPinNum and dataPinNum?
Also add command and link about how the pin number works.
https://godoc.org/github.com/stianeikeland/go-rpio#Pin